### DISCUSSION TITLE
[Skia/WPF] AutomationProperties.AutomationId not working

### DISCUSSION THREAD

**Discussion by bstrother1** [unoplatform/uno#9213] [Q&A] [answered]
Am I missing something here? : 
`  <TextBlock AutomationProperties.AutomationId="ATextBlock" Text="Hello World"/>`

but searching for it here:
   `Assert.AreEqual(textLabel , this.AppSession.FindElementByAccessibilityId("ATextBlock").Text);`

throws an Element not found error. 
Is this a bug? To add onto this xaml has:  Property 'AutomationId' is not attachable to elements of type 'TextBlock' when hovering over it. Same things happens on any element with AutomationId attached.

**Comment by nickrandolph** [comment, marked as answer]
There was an issue (see https://github.com/unoplatform/uno/issues/9167) relating to AutomationId on Skia/Wpf - can you try updating to the latest dev package of uno to see if this resolves the issue?

**Reply by bstrother1** [reply]
I am on the latest 4.5.0-dev.63 .

**Reply by francoistanguay** [reply]
This is for UI automation testing right? 

If so, AFAIK, we don't have Skia/WPF support for Uno UI Testing yet.

**Reply by bstrother1** [reply]
Nope, Appium. Can be found here https://www.telerik.com/blogs/get-your-wpf-apps-automated-appium . Already was aware Uno.UI only works for Android and IOS. Is there incompatibility between these two? As far as I understand a WPF.exe built by Uno should act like a normal WPF application and should be able to have elements locatable by Appium.

**Reply by francoistanguay** [reply]
Pretty sure Appium will see Uno as a blackbox. We're using WPF solely as a host to render our Skia canvas. So Appium wouldn't have any knowledge of the Uno-based Visual tree. Will check further with the team.

**Reply by francoistanguay** [reply]
More info here: https://github.com/unoplatform/Uno.UITest#unouitest-for-uno-platform

**Reply by bstrother1** [reply]
What am I meant to look at in this documentation?

**Reply by francoistanguay** [reply]
Confirmation that Skia/WPF isn't supported and which platforms are supported at this point. This is clearly something we'll investigate in the future and will add to our roadmap.

**Reply by bstrother1** [reply]
Should a ticket be made?
